beta/podman/README.md

```diff
@@ -0,0 +1,244 @@
+
+  ```dotenv
+  ...
+  # (GOOGLE WORKSPACE) the options below are intended for users of our Google Workspace integration
+  # OP_WORKSPACE_CREDENTIALS should be base64-encoded Google Service Account credentials JSON
+  OP_WORKSPACE_CREDENTIALS=/run/secrets/workspace-credentials
+  # OP_WORKSPACE_SETTINGS is a base64-encoded string containing Workspace settings JSON for your bridge
+  OP_WORKSPACE_SETTINGS=/run/secrets/workspace-settings
+  ...
+  ```
+
+5. Run the `deploy.sh` script by using the command below
+
+  ```sh
+  ./deploy.sh --file podman.http.yaml --file podman.gw.yaml
+  ```
+
+
+## Test your SCIM bridge
+
+Run this command to view logs from the service for the SCIM bridge container:
+
+```sh
+podman logs scim
+```
+
+Your **SCIM bridge URL** is based on the fully qualified domain name of the DNS record created in [Get started](#get-started). For example: `https://scim.example.com`. Replace `mF_9.B5f-4.1JqM` with your bearer token and `https://scim.example.com` with your SCIM bridge URL to test the connection and view status information.
+
+```sh
+curl --silent --show-error --request GET --header "Accept: application/json" \
+  --header "Authorization: Bearer mF_9.B5f-4.1JqM" \
+  https:/scim.example.com/health
+```
+
+<details>
+<summary>Example JSON response:</summary>
+
+```json
+{
+  "build": "209031",
+  "version": "2.9.3",
+  "reports": [
+    {
+      "source": "ConfirmationWatcher",
+      "time": "2024-04-25T14:06:09Z",
+      "expires": "2024-04-25T14:16:09Z",
+      "state": "healthy"
+    },
+    {
+      "source": "RedisCache",
+      "time": "2024-04-25T14:06:09Z",
+      "expires": "2024-04-25T14:16:09Z",
+      "state": "healthy"
+    },
+    {
+      "source": "SCIMServer",
+      "time": "2024-04-25T14:06:56Z",
+      "expires": "2024-04-25T14:16:56Z",
+      "state": "healthy"
+    },
+    {
+      "source": "StartProvisionWatcher",
+      "time": "2024-04-25T14:06:09Z",
+      "expires": "2024-04-25T14:16:09Z",
+      "state": "healthy"
+    }
+  ],
+  "retrievedAt": "2024-04-25T14:06:56Z"
+}
+```
+
+</details>
+<br />
+
+To view this information in a visual format, visit your SCIM bridge URL in a web browser. Sign in with your bearer token, then you can view status information and download container log files.
+
+## Connect your identity provider
+
+Use your SCIM bridge URL and bearer token to [connect your identity provider to 1Password SCIM Bridge](https://support.1password.com/scim/#step-3-connect-your-identity-provider).
+
+## Update your SCIM bridge
+
+👍 Check for 1Password SCIM Bridge updates on the [SCIM bridge releases notes website](https://releases.1password.com/provisioning/scim-bridge/).
+
+To upgrade your SCIM bridge, `git pull` the latest versions from this repository. Then re-apply the `.yml` file. For example:
+
+```bash
+cd scim-examples/
+git pull
+cd beta/podman/
+podman-compose up --build -d
```

Comment:
@ag-rdoucette Would this have to change to redeploying the bridge using deploy.sh?